mods/sample-nginx.config

```diff
@@ -88,6 +88,11 @@ server {
     rewrite ^/(.*) /index.php?pagename=$uri last;
   }
 
+  # add rewrite needed by install process
+  if (!-f $request_filename) {
+      rewrite ^/fd/(.+)$ /fd/index.php?q=$1 last;
```

Comment:
What is this ```/fd``` folder?

Reply:
Cargo-culted from https://github.com/friendica/friendica/issues/868